src/core/include/cesium/omniverse/GltfUtil.h

```diff
@@ -27,6 +27,8 @@ enum class AlphaMode : int {
     BLEND = 2,
 };
 
+enum class OverlayType { IMAGERY = 0, POLYGON = 1 };
```

Comment:
I've added another enum since this comment while separating data-providing from rendering, and `OverlayType::IMAGERY` is now gone:

```
enum class OverlayType { ION = 0, POLYGON = 1 };
enum class OverlayRenderPipe { OVERLAY = 0, CLIP = 1 };
```